### PR TITLE
Feature/add pytest to jenkins

### PR DESCRIPTION
**Pull request by kevinyeungxmos**

### CODE REVIEW

**Comment by xross** [comment]
I would run the tests on a single process until #230 is resolved.

**Review by xross — CHANGES_REQUESTED** [review]
I would run the tests on a single process until #230 is resolved.

**Comment by xross** [comment]
@kevinyeungxmos Currently fails with: ModuleNotFoundError: No module named 'filelock'

File: Jenkinsfile [hunk 1]
```diff
@@ -37,11 +37,15 @@ pipeline {
...
     }
-    // stage('Tests') {
-    //   steps {
-    //     runXmostest("${REPO}", 'tests')
-    //   }
-    // }
+    stage('Tests') {
+      steps {
+        viewEnv() {
+          dir("${REPO}/tests") {
```
Comment (danielpieczko) [inline Jenkinsfile:43]: To use the python virtualenv that includes the dependencies in requirements.txt, you need a withVenv() closure here, along with the viewEnv() and dir() you already have.

**Pull request closed without merging**